UPGRADING.md

```diff
@@ -3,6 +3,30 @@ Upgrading Grape
 
 ### Upgrading to >= 0.14.0
 
+#### Changes to behavior of `after` method of middleware on error
+
+The `after` method of the middleware is now called on error.
+The following code would work correctly.
+
+```ruby
+class CustomMiddleware < Grape::Middleware::Base
+  def after
+    puts "Yay!"
+  end
+end
+
+class API < Grape::API
+  use CustomMiddleware
+
+  get { raise StandardError }
+end
+
+mock = Rack::MockRequest.env_for(?/)
```

Comment:
This is very spec-y, remove. Rewrite the example to be more about a real world API, and maybe return a proper rack response in `after`. In this example it's unclear what happen.

On this note we also need a README update about the behavior of `after`. Shoot, I keep forgetting things.... lmk when you had enough of my comments and I'll do the rest ;)